lib/swagger/jsonFullResolvedLoader.ts

```diff
@@ -0,0 +1,248 @@
+import { dirname as pathDirname } from "path";
```

Comment:
What's the main diff of this file comparing to jsonLoader? It would be better to call the same code rather than having two copies.